webrecorder/browsertrix
# docs: Adds information about 1.6 features to documentation

Partially addresses #922, closes #1083 

### Changes

- Adds note in style guide about referencing actions in the app
- Adds page for Browser Profiles
  - Adds callout for uploads in the context of combining items from multiple sources
- Adds page for Collections
- Adds page for Crawl Workflows
- Updates index to link to new dedicated Crawl Workflow page in addition to the Crawl Workflow Setup page
- Updates Org Settings page action styling in accordance with new rules
- Updates Crawl Workflow Setup page with links to the new pages and a hierarchy fix for the first item
- Updates user guide navigation with a new section for crawling related items

### Testing

1. Change to the branch, run mkdocs serve
2. Navigate to http://127.0.0.1:8000/user-guide/
3. Read through the new sections and check if I missed anything important for this pass! :)
